### DISCUSSION TITLE
No available domains to choose when importing from Google Analytics

### DISCUSSION THREAD

**Discussion by Debcharon** [plausible/analytics#3137] [Self-Hosted Support]
No available domains to choose when importing from Google Analytics, I've already enabled Google Search Console API, Analytics Reporting API and Google Analytics API in advanced, but I can bind Google account for Google Search Console integration.

How to solve this problem, thanks. images are below.

![image](https://github.com/plausible/analytics/assets/107681877/898a690f-bbca-439a-a5c2-24306dc73be1)

![image](https://github.com/plausible/analytics/assets/107681877/572e50e9-d09b-4330-83f1-069106113089)

![image](https://github.com/plausible/analytics/assets/107681877/43a1c541-ab8d-42a8-9818-0469bc6a03f7)

![image](https://github.com/plausible/analytics/assets/107681877/1f8abc82-acc0-4a56-998e-8f82f2d1ec99)

![image](https://github.com/plausible/analytics/assets/107681877/c6056363-aba1-4030-a46c-69910872c74c)

**Comment by boy51** [comment]
Hi did you find solution for this?

**Reply by Debcharon** [reply]
Unfortunately, till now I didn't find solution that can solve, even tried find "Enable Users Metric in Reporting" in Google Analytics admin. I cannot find "Enable Users Metric in Reporting" in GA.

https://plausible.io/docs/google-analytics-import#my-import-has-failed
